### ISSUE TITLE
Selenium returns element isDisplayed when the element is not Visible

### ISSUE THREAD

**Issue by bebef1987** [SeleniumHQ/selenium#3250]
OS:  
Windows7
Selenium Version:  
3.0.1
geckodriver 0.11.1
ChromeDriver 2.25.426923 (0390b88869384d6eb0d5d09729679f934aab9eed)

Browser:  
Chrome  55.0.2883.87 m
Firefox 50.0.2

## Expected Behavior -

3. Element ".app__accessories__group #js__010-11430-00"  should not be detected as Displayed

## Actual Behavior -

3. Element ".app__accessories__group #js__010-11430-00" is returned as Displayed

## Steps to reproduce -
1. Open https://buy.garmin.com/en-US/US/p/134491#accessories
2. Inspect element: .app__accessories__group #js__010-11430-00
3. Check is element isDisplayed()

Run:
```Java
		driver = new FirefoxDriver();
//		driver = new ChromeDriver();

		driver.get(
				"https://buy.garmin.com/en-US/US/p/134491#accessories");
		Thread.sleep(5000);

		WebElement webEl = driver.findElement(
				By.cssSelector(".app__accessories__group #js__010-11430-00"));
		System.out.println(webEl.isDisplayed());

		driver.quit();
```

**Comment by lmtierney** [comment]
[Non-dev link...](https://buy.garmin.com/en-US/US/p/134491#accessories)

**Comment by bebef1987** [comment]
@lmtierney Updated the code sorry  forgot about that

**Comment by lmtierney** [comment]
Does it work in chromedriver?

**Comment by bebef1987** [comment]
Nope tested it in chrome and it still fails

```
Starting ChromeDriver 2.25.426923 (0390b88869384d6eb0d5d09729679f934aab9eed) on port 43836
Only local connections are allowed.
Dec 14, 2016 4:43:16 PM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Attempting bi-dialect session, assuming Postel's Law holds true on the remote end
Dec 14, 2016 4:43:17 PM org.openqa.sel
4:43:17 PM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: OSS
Is element displayed: true

```

**Comment by lmtierney** [comment]
I suppose this would have been another good example [for the talk at seleniumconf](https://www.youtube.com/watch?v=hTa1KI6fQpg)

**Comment by bebef1987** [comment]
@AutomatedTester  this looks like something you would be interested

**Comment by kavalerchik** [comment]
I also have this problem in my tests but **only** in Firefox.
WIndows 10, Selenium 3.0.1, Gecko driver 0.11.1, chrome driver 2_0
Firefox ver 55.0.2, Chrome ver 60.0.3xxx

**Comment by barancev** [comment]
Fixed by 1374b338c081ee86ab64ef02024d4f07edecf1d4
